src/ResourceManager/EventHub/Commands.EventHub/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.EventHub" version="2.2.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.EventHub" version="2.8.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
```

Comment:
@v-Ajnava same comment about removing additional references in this file

Reply:
resolved